common/util.ts

```diff
@@ -15,3 +15,7 @@ export const getMapLanguage = () => {
   const languageCode = parts.length > 1 ? parts[0] : OSDLanguage;
   return OSD_LANGUAGES.includes(languageCode) ? languageCode : FALLBACK_LANGUAGE;
 };
+
+export function isEscapeKey(e: KeyboardEvent) {
+  return e.key === 'Escape';
```

Comment:
How about use keyCode 27 instead of `'Escape'` for compatible across different browsers?
```
export function isEscapeKey(e: KeyboardEvent) {
  return e.keyCode === 27;
}
```

Reply:
It is deprecated. https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode